### DISCUSSION TITLE
Content Security errors after switching from object storage to local storage

### DISCUSSION THREAD

**Discussion by vengefulpunk** [mastodon/mastodon#30915] [Q&A]
I have a ton of local storage available and decided to switch away from object storage and go to local. This instance was initially object storage from the first time I spun it up. Everything is working fine except in console I am receiving Content Security Policy errors. 
![loaderrors](https://github.com/mastodon/mastodon/assets/7458716/841df347-f5c8-445a-988d-b6b70f3dfd9a)

I tried finding if something was still set to object storage for a security policy but could not find any settings or mention of that in any of the files or in the admin backend. 

I am assuming the 404 errors below may be related but can dive into that if fixing the content security policy does not fix it.

**Comment by ClearlyClaire** [comment]
I could not reproduce any of this visiting your domain; are you still experiencing these issues?

The Content Security Policy violation is a bit surprising because I can't think of any usage of inline styles in Mastodon.

The 404 seem indicative of a misconfigured server, that does not return the expected assets, but since I'm not experiencing those, I assume this wasn't the issue, or you already fixed it.

Another possibility might be the service worker somehow still using old values. If you can reproduce the issue, can you still reproduce it in e.g. a new private browser session?
